src/orbit-provider.ts

```diff
@@ -0,0 +1,297 @@
+              let body = msg.q.body;
+              let json: any = await postJSON(makeUrl(suffUrl, msg.q), makeConfig(body))
+              let data = json.data
+              let list = entize(data)
+              return list
+            },
+          }
+        }
+      },
+      get_member: {
+        cmd: {
+          list: {
+            action: async function(this: any, entize: any, msg: any) {
+              let suffUrl = getWorkspace() + '/members/' + msg.q.idMember
+              let json: any = await getJSON(makeUrl(suffUrl, msg.q), makeConfig())
+              let data = json.data
+              let list = entize(data)
+              return list
+            },
+          },
+        }
+      },
+      find_member_by_identify: {
+        cmd: {
+          list: {
+            action: async function(this: any, entize: any, msg: any) {
+              let suffUrl = getWorkspace() + '/members/find'
+              let json: any = await getJSON(makeUrl(suffUrl, msg.q), makeConfig())
+              let data = json.data
+              let list = entize(data)
+              return list
+            },
+          },
+        }
+      },
+      list_member_by_organization: {
+        cmd: {
+          list: {
+            action: async function(this: any, entize: any, msg: any) {
+              let suffUrl = getWorkspace() + '/organizations/' + msg.q.idOrganization + '/members'
+
+              let json: any = await getJSON(makeUrl(suffUrl, msg.q), makeConfig())
+              let data = json.data
+              let list = data.map((data: any) => entize(data))
+              return list
+            },
+          },
+        }
+      },
+      update_member: {
+        cmd: {
+          save: {
+            action: async function(this: any, entize: any, msg: any) {
+              let suffUrl = getWorkspace() + '/members/' + msg.q.idMember
+              let body = msg.q.body;
+              let res: any = await putJSON(makeUrl(suffUrl, msg.q), makeConfig(body))
+              
+              let list = entize(res)
+              return list
+            },
+          }
+        }
+      },
+      create_or_update_member: {
+        cmd: {
+          save: {
+            action: async function(this: any, entize: any, msg: any) {
+              let suffUrl = getWorkspace() + '/members'
+              let body = msg.q.body;
+              let json: any = await postJSON(makeUrl(suffUrl, msg.q), makeConfig(body))
+              let data = json.data
+              let list = entize(data)
+              return list
+            },
+          }
+        }
+      },
+    }
+  })
+
+
+
+  seneca.prepare(async function(this: any) {
+    let res =
+      await this.post('sys:provider,get:keymap,provider:orbit')
+
+    if (!res.ok) {
+      throw this.fail('keymap')
+    }
+
+    this.shared.headers = {
+      Authorization: 'Bearer ' + res.keymap.key.value
+    }
+
+    setWorkspace(res.keymap.workspace.value)
```

Comment:
refactor this to use `this.shared.workspace`

Reply:
ok, swapped